src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/create")
```

Comment:
to create a car u use "/cars/add", to create a driver u use "/drivers/create" ~?

Reply:
Agree, let's make it consistent, use one naming strategy across all the controllers.